docs/vm-installation.md

```diff
@@ -0,0 +1,115 @@
+# Installing antrea-agent on VM
+
+`antrea-agent` can be run on Linux/Windows VMs. Below are the steps needed to configure and run
+antrea-agent on VMs
+
+## Prerequisites on Kubernetes cluster
+
+1. Enable ExternalNode feature on the `antrea-controller`
+2. Create a Namespace for `antrea-agent`
+
+```bash
+kubectl create ns vm-ns
+```
+
+3. Create a ServiceAccount, ClusterRole and ClusterRoleBinding for `antrea-agent`
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/feature/externalnode/build/yamls/externalnode/vm-agent-rbac.yml
+```
+
+4. Create antrea-agent.kubeconfig file to access K8S API service
+
+```bash
+export CLUSTER_NAME=<cluster-name>
+export SERVICE_ACCOUNT=<service-account-name>
+APISERVER=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$CLUSTER_NAME\")].cluster.server}")
+TOKEN=$(kubectl -n vm-ns get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='$SERVICE_ACCOUNT')].data.token}"|base64 --decode)
+kubectl config --kubeconfig=antrea-agent.kubeconfig set-cluster kubernetes --server=$APISERVER --insecure-skip-tls-verify=true
+kubectl config --kubeconfig=antrea-agent.kubeconfig set-credentials antrea-agent --token=$TOKEN
+kubectl config --kubeconfig=antrea-agent.kubeconfig set-context antrea-agent@kubernetes --cluster=kubernetes --user=antrea-agent
+kubectl config --kubeconfig=antrea-agent.kubeconfig use-context antrea-agent@kubernetes
+```
+
+5. Create antrea-agent.antrea.kubeconfig file to access `antrea-controller` API service
```

Comment:
Why it is named "antrea-agent.antrea.kubeconfig", but not "antrea-agent.kubeconfig" (I was the latter was the name)?

Reply:
Add `` around antrea-agent.antrea.kubeconfig: `antrea-agent.antrea.kubeconfig`

Reply:
Why it is called "API service"? Should we use "API" or "API server"?

Reply:
Changed it to API server

Reply:
I think we have references to similar names for running antrea-agent manually for container cases. 
@wenyingd could you please share your thoughts

Reply:
There are two different kubeconfig files: one is used to connect to APIServer with name `antrea-agent.kubeconfig`, and the other is used to connect to Antrea Controller with name "antrea-agent.antrea.kubeconfig". The middle word "antrea" in second file is actually the cluster name.

For container case, if Antrea Agent is not running inside a Pod, e.g. as a process/Service on Windows, these two kubeconfig files are also needed.